server/src/main/java/org/eclipse/hono/server/MetricConstants.java

```diff
@@ -0,0 +1,32 @@
+package org.eclipse.hono.server;
+
+/**
+ * Constants/metric names
+ */
+public class MetricConstants {
+
+    public static String metricNameDownstreamConnections() {
+        return "hono.server.connections.downstream";
+    }
+
+    public static String metricNameDownstreamLinkCredits(String address) {
+        return "hono.server.link.downstream.credits." + address;
```

Comment:
In such simple concatenation with only one postfix, I find the simple string concatenation more readable. But String.format() might have a better performance. Or does it look clearer for you?